### PR TITLE
Added new config Options

### PR DESCRIPTION
**Pull request by niksingh710**
This allows the user to put any glyphs in their config to represent their to-do status. Sources to get glyphs [nerd font](https://www.nerdfonts.com/)

The new config may look like this:
```json
{
  "user_name": "Nikhil Singh",
  "initial_setup_done": true,
  "tasks": [],
  "disable_line": false,
  "disable_quotes": false,
  "disable_greeting": false,
  "done_icon": "",
  "notdone_icon": "",
  "time_format_24h": false
}
```
This will not break the user's previous config as the options `done_icon` and `notdone_icon` are optional.

### CODE REVIEW

File: please/please.py [hunk 1]
```diff
@@ -332,6 +332,8 @@ def setup() -> None:
     config["disable_greeting"] = False
     config["time_format_24h"] = False
     config["last_reminder"] = None
+    conig["done_icon"] = "✅"
```
Comment (NayamAmarshe) [inline please/please.py:335]:
```suggestion
    config["done_icon"] = "✅"
```

File: please/please.py [hunk 2]
```diff
@@ -332,6 +332,8 @@ def setup() -> None:
...
+    conig["notdone_icon"] = "❌"
```
Comment (NayamAmarshe) [inline please/please.py:336]:
```suggestion
    config["notdone_icon"] = "❌"
```

**Review by NayamAmarshe — COMMENTED** [review]
Thanks for the PR. Did you test the changes too?

**Comment by niksingh710** [comment]
> Thanks for the PR. Did you test the changes too?

Yes tested the changes working perfectly fine.

File: please/please.py (hunk 1 above)
Comment (niksingh710) [reply please/please.py:335]: 😅 I guess I missed this typo as I had those variables in config.
Will update it.

**Comment by niksingh710** [comment]
Now it is ready to merge. @NayamAmarshe

**Comment by NayamAmarshe** [comment]
Thanks for the fix.

**Pull request merged**